src/Mystira.App.PWA/Pages/CharacterAssignmentPage.razor.css

```
@@ -589,51 +589,119 @@ html[data-theme="dark"] .character-card .text-muted {
     }
 }
 
-html[data-theme="dark"] .btn-media {
-    background: rgba(99, 102, 241, 0.15);
-    border-color: rgba(99, 102, 241, 0.4);
-    color: #a5b4fc;
+/* Dark mode explicit styles - using ::deep to escape component scope */
+::deep html[data-theme="dark"] .btn-media {
```

Biome (2.3.13)

[error] Unexpected unknown pseudo-elements: deep

See MDN web docs for more details.
Use a known pseudo-elements instead, such as:


(lint/correctness/noUnknownPseudoElement)